src/api/filesystem/sync_io.rs

```diff
@@ -896,6 +896,14 @@ pub trait FileSystem {
     fn notify_reply(&self) -> io::Result<()> {
         Err(io::Error::from_raw_os_error(libc::ENOSYS))
     }
+
+    /// Remap the external IDs in context to internal IDs.
+    fn id_remap(
+        &self,
+        ctx: &mut Context
+    ) -> io::Result<()> {
+        Err(io::Error::from_raw_os_error(libc::ENOSYS))
```

Comment:
Let's just return OK() here to avoid breaking existing file system implementations. And it makes sense as id_remap is not a MUST-have feature.